components/RootHomePage/Shop.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+import LinkButton from '../shared/LinkButton';
+import ContentSection from '../shared/ContentSection';
+
+import { SlimCenteredH2 } from '../shared/StandardStyles';
+import { below } from '../../utilities';
+
+const Main = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin: auto;
+  justify-content: center;
+  align-items: center;
+
+  ${below.med`
+    flex-direction: column;
+  `};
+`;
+
+const sharedBearStyles = css`
+  height: 40rem;
+
+  ${below.med`
+    height: 20rem;
+  `};
+`;
+
+const LeftBear = styled.img`
+  ${sharedBearStyles};
+  margin-right: 7rem;
+
+  ${below.med`
+    margin-right: 0;
+    margin-bottom: 4rem;
+  `};
+`;
+
+const RightBear = styled.img`
+  ${sharedBearStyles}
+  transform: scaleX(-1);
+  margin-left: 7rem;
+
+  ${below.med`
+    margin-left: 0;
+    margin-top: 4rem;
+  `};
+`;
+
+const Shop = ({ className }) => {
+  return (
+    <ContentSection className={className}>
+      <Main>
+        <LeftBear src="./images/bear_pink_hoodie.png" />
+        <div>
+          <SlimCenteredH2>Get That Gear</SlimCenteredH2>
+          <h4>That Store Has Your Favorite Merch</h4>
+          <p className="blurb">
+            Looking for a new favorite THAT T-Shirt? What about a THAT Hoodie?
+            THAT Store is now open and has the latest THAT merch waiting for
+            you.
+          </p>
+          <LinkButton
+            label="Go to THAT Store"
+            href="http://wwww.thatconference.store"
```

Comment:
One too many w's - you can just make it thatconference.store because it 301s into the naked domain.